diff --git a/ldap_auth_provider.py b/ldap_auth_provider.py
index 96296b6..d6101b8 100644
--- a/ldap_auth_provider.py
+++ b/ldap_auth_provider.py
@@ -21,7 +21,8 @@ from typing import Any, Dict, Iterable, List, Optional, Tuple, Union
 import ldap3
 import ldap3.core.exceptions
 import synapse
-from pkg_resources import parse_version
+# Fix parse_version import Applied
+from packaging.version import parse as parse_version
 from synapse.module_api import ModuleApi
 from synapse.types import JsonDict
 from twisted.internet import threads
